### PR TITLE
obsolete absence epilepsy

### LINKED ISSUE #8468: [Obsolete] MONDO:0850093 'absence epilepsy'
**Mondo term (ID and Label)**
MONDO:0850093 'absence epilepsy'

**Reason for deprecation**
This is a phenotype. Discussed at [epilepsy workshop](https://www.youtube.com/watch?v=W_WC7QZsL0M) around minute 18.

**Suggested term to consider**
[HPO](https://hpo.jax.org/browse/term/HP:0011147)

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
close #8468

### CODE REVIEW

**Review by sabrinatoro — APPROVED** [review]
approving and merging
